src/cmd/commit.ts

```diff
@@ -1,4 +1,4 @@
-import { copyFileSync, existsSync } from 'fs'
```

Comment:
Still need to review this file

Reply:
Done. 
IMO bootstrapGit is a way too complex function.
Splitting it into `hasCommits()` and `syncFilesWithLocalRepo` and `initGit()` functions would significantly increase readability.

Reply:
don't exaggerate. It is readable and needs to be structured that way. Not going to refactor as it won't result in better code.